woocommerce/woocommerce
# Tweak - Added links to shipping methods configuration in shipping zones.

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

1. Create shipping zone
2. Add shipping method 
3. Go to Shipping zones - see link to shipping method config on shipping method title.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Added links on shipping method titles on shipping zones configuration page.

![image](https://user-images.githubusercontent.com/8124521/89573051-4bca5a00-d82a-11ea-9425-1ff63b7dd03b.png)
